views/headlines.html

```diff
@@ -0,0 +1,18 @@
+<h1>{{pageTitle}}</h1>
+<ul class="headlines">
+	{{#each results}}
```

Comment:
nailed it. #each comes in very handy for this. We use it often

Reply:
I found by using the `{{#each <object }}` to be really pleasant to work with.